confignode/src/main/java/org/apache/iotdb/confignode/persistence/RegionInfoPersistence.java

```diff
@@ -51,8 +49,8 @@ public class RegionInfoPersistence {
 
   // Region read write lock
   private final ReentrantReadWriteLock regionReadWriteLock;
-  // Map<ConsensusGroupId, RegionReplicaSet>
-  private final Map<ConsensusGroupId, RegionReplicaSet> regionMap;
+  // Map<TConsensusGroupId, TRegionReplicaSet>
```

Comment:
what's this?

Reply:
A Map that describes the location of SchemaRegions and DataRegions.

Reply:
I mean this comments seems useless.

Reply:
fixed.